Module/STIG/Functions.Checklist.ps1

```diff
@@ -47,7 +47,7 @@ function New-StigCheckList
         [string]
         $XccdfPath,
 
-        [Parameter(Mandatory = $true)]
+        [Parameter(Mandatory = $false)]
```

Comment:
[Style guidelines](https://github.com/PowerShell/DscResources/blob/master/StyleGuidelines.md#correct-format-for-parameter-block) say non-mandatory parameters shouldn't us a mandatory decoration.

Reply:
Fixed.